lib/bamboo/mailer.ex

```diff
@@ -84,6 +81,9 @@ defmodule Bamboo.Mailer do
         Use deliver_now/1 to send right away, or deliver_later/1 to send in the background.
         """
       end
+
+      otp_app = Keyword.fetch!(opts, :otp_app)
```

Comment:
I think a newline between this and `defp config` would be nice. What do you think?